packages/eddsa-pcd/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@pcd/eddsa-pcd",
+  "version": "0.1.0",
+  "license": "GPL-3.0-or-later",
```

Comment:
could you make sure this license is appropriate for the dependencies we're using - i'm assuming this needs to be gpl because of `circomlib`

Reply:
`circomlibjs` is GPL-3.0. The other PCD packages are `GPL-3.0-or-later`.